packages/lib-classifier/src/components/Classifier/components/SubTaskPopup/SubTaskPopup.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import counterpart from 'counterpart'
+import { Box } from 'grommet'
+import { Rnd } from 'react-rnd'
+import { inject, observer } from 'mobx-react'
+import {} from 'prop-types'  // TODO
+
+import SingleChoiceTask from '@plugins/tasks/SingleChoiceTask'
+
+import styled from 'styled-components'  // TODO: check what's the best way to style this component
+import zooTheme from '@zooniverse/grommet-theme'
+
+// EXPERIMENTAL
+// ----------------
+import taskRegistry from '@plugins/tasks'
+const StyledBox = styled(Box)`
+  border: 2px solid ${zooTheme.global.colors.brand}
```

Comment:
Box has a border prop, so this shouldn't be necessary? https://v2.grommet.io/box#border